ui/cypress/integration/tests.js

```diff
@@ -25,5 +25,9 @@ describe("Basic tests", () => {
 
     cy.get("input[name='queries-mode']").click();
     cy.contains("SELECT *");
+
```

Comment:
question: Does this file only for the testing? When should I consider to add it?

Reply:
This is an integration test that runs an actual browser to render the page and takes actions on it. Eventually we'll want to expand this into multiple tests for different parts of the UI. It would good for you to add a simple test of the demographics too.